### PR TITLE
added another sentence

### PR DESCRIPTION
**Pull request by charlesvickery**

### CODE REVIEW

**Comment by charlesvickery** [comment]
You misspelled a word in there, partner.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # mython
 ##an introductory project in Python to make me a Pythonista.
 
+Hi Milton!!
```
Comment (beyang) [inline README.md:4]: Is this correct?

**Comment by beyang** [comment]
hi there

**Comment by beyang** [comment]
LGTM

**Pull request merged**